packages/messages/src/browser/notifications.ts

```diff
@@ -29,6 +30,7 @@ export interface Notification {
 
 export class Notifications {
 
+    private closeTimer: any;
```

Comment:
it's `number | undefined` if you use `window.setTimeout` below.

Reply:
1. I think this should go into the scope of `createNotificationElement`
2. we can avoid use `any` here, as already commented

Reply:
Move the closeTimer to createNotificationElement